src/stitching/makeRemoteExecutableSchema.ts

```diff
@@ -142,6 +161,37 @@ function createResolver(fetcher: Fetcher): GraphQLFieldResolver<any, any> {
   };
 }
 
+function createSubscriptionResolver(link: ApolloLink): ResolverFn {
+  return (root, args, context, info) => {
+    const fragments = Object.keys(info.fragments).map(fragment => info.fragments[fragment]);
+    const document = {
+      kind: Kind.DOCUMENT,
+      definitions: [info.operation, ...fragments],
+    };
+
+    const operation = {
+      query: document,
+      variables: info.variableValues,
+      context: { graphqlContext: context },
+    };
+    const observable = execute(link, operation);
+
+    const pubSub = new PubSub();
```

Comment:
I think users of this should supply their own pubsub right?

Reply:
Though i understand the usage here

Reply:
:+1: on providing pubsub.

Reply:
Any API design suggestions for this?